Index: src/main/java/com/inyong/section06/users/controller/MoneyController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/inyong/section06/users/controller/MoneyController.java b/src/main/java/com/inyong/section06/users/controller/MoneyController.java
new file mode 100644
--- /dev/null	(date 1740616400205)
+++ b/src/main/java/com/inyong/section06/users/controller/MoneyController.java	(date 1740616400205)
@@ -0,0 +1,4 @@
+package com.inyong.section06.users.controller;
+
+public class MoneyController {
+}
